distribution/ddf-common/src/main/resources/etc/log4j2.config.xml

```diff
@@ -75,6 +75,16 @@
                 <AppenderRef ref="securityBackup"/>
             </Failovers>
         </Failover>
+
+        <RollingFile name="solr" append="true"
+                     fileName="${sys:karaf.data}/log/solr.log"
+                     filePattern="${sys:karaf.data}/log/ingest_error.log-%d{yyyy-MM-dd-HH}-%i.log.gz">
```

Comment:
ingest_error.log should be solr.log on this line.  Looks like a copy and paste mistake.